BILL_OF_MATERIALS.md

```diff
@@ -0,0 +1,43 @@
+# Bill Of Materials (BOM)
+
+
+## Alexander Koch arms
+
+### Follower and Leader arms
+
+TODO
+
+### Leader arm only
+
+| Part | Amount | Cost/p US | Buy US | Cost/p EU | Buy EU | Cost/p UK | Buy UK |
+|---|---|---|---|---|---|---|---|
+| Dynamixel XL430-W250-T |  2 | $50 | [Robotis](https://www.robotis.us/dynamixel-xl430-w250-t) | 57-61€ | [MyBotShop](https://www.mybotshop.de/DYNAMIXEL-XL430-W250-T)-[GenRobots](https://www.generationrobots.com/en/402823-dynamixel-xl430-w250-t-servomotor.html) | £47 | [RoboSavvy](https://robosavvy.co.uk/dynamixel-xl430-w250-t.html)
+| Dynamixel XL330-M288-T |  4 | $24  | [Robotis](https://www.robotis.us/dynamixel-xl330-m288-t) | 40-46€ | [MyBotShop](https://www.mybotshop.de/DYNAMIXEL-XL330-M288-T)-[GenRobots](https://www.generationrobots.com/en/403817-dynamixel-xl330-m288-t-servo-motor.html) | £27.10 | [RoboSavvy](https://robosavvy.co.uk/robotis-dynamixel-xl330-m288-t.html) |
+| XL330 Frame and Idler Wheel 4pcs set |  1 | $10 | [Robotis](https://www.robotis.us/fpx330-h101-4pcs-set) | 12€ | [GenRobots](https://www.generationrobots.com/en/403860-FPX330-H101-hinge-frame-and-idler-set-dynamixel-xl330.html) | | |
+| XL430 Idler Wheel set | 1 | $7 | [Robotis](https://www.robotis.us/hn11-i101-set) | 9€ | [GenRobots](https://www.generationrobots.com/en/403206-hn11-i101-horn-set.html) | | |
+| Waveshare Serial Bus Servo Driver Board | 1 | $10 | [Amazon](https://a.co/d/7C3RUYU) | 6€ | [Eckstein](https://eckstein-shop.de/WaveShare-Serial-Bus-Servo-Driver-Board-for-ST-SC-Serial-Bus-Servos-EN) | | |
+| Voltage Reducer 6pcs set | 1 | $10 | [Amazon](https://a.co/d/cy02ADW) | 6€ | [Garmade](https://garmade.com/en-fr/products/ldtr-wg0178-lm2596-power-step-down-voltage-regulator-module-voltmeter-display) | | |
+| 12V Power Supply | 1 | $12 | [Amazon](https://a.co/d/40o8uMN) | 15-36€ | [Amazon](https://www.amazon.fr/LEDMO-Alimentation-Adaptateur-Transformateurs-Chargeur/dp/B07PGLXK4X)-[GenRobots](https://www.generationrobots.com/en/400866-smps-charger-for-bioloid-and-dynamixel-robotis.html) | | |
+| Jumper Wires 3*40 pcs set (M-M, M-F, F-F) | 1 | $7 | [Amazon](https://a.co/d/hQfk2cb) | 9€ | [Amazon](https://www.amazon.fr/AZDelivery-Jumper-Cavalier-C%C3%A2ble-Arduino/dp/B074P726ZR) |  | |
+| Table Clamp | 1 | $6 | [Amazon](https://a.co/d/4KEiYdV) | n/a | n/a | | |
```

Comment:
Nit: I am using two small plastic clamps such as this: https://www.amazon.com/Mr-Pen-Carpenter-Clamp-6inch/dp/B092L925J4/ref=sr_1_5?crid=2RJXX25G9JMMF&dib=eyJ2IjoiMSJ9.IsVfHXbaTzSziJqlIc4Sgz3n_LiHYa3wMfXVzwRvXZek8C7hG_B9-ZCbnklZ0r80xjP8PHTM98uPPWKNPYKiue3UuKM8PS7SrtIgBHserGUjnx_kmMejOcsAZqFZDVQz3i9uc5oPBhNZudbTbKUfmsXVVLfHoNU0N5MI2bQ3VUAdwuvnab51yLjWlK73sQTiarH8xA0NQRa2pIxkH73iuH_gbLHu7IvlYtaMouXq594kMWf5E6tq0MvZbA7Ac4NcFEZWaz5dujTni2sXKjn14pD_HY43PJivBJygQkT2DVQ.wAFrAx0a8-ckm74LcqHyR4yqpA0Y9btgszhcdjOKUx8&dib_tag=se&keywords=plastic%2Bclamp&qid=1720538954&refinements=p_36%3A-900&rnid=1243644011&s=hi&sprefix=plastic%2Bclamp%2Ctools%2C89&sr=1-5&th=1

It is not necessary to change this but if you are building both the follower and leader arm, it would be more cost effective. Maybe just something to keep in mind if we create a BoM for both.